### ISSUE TITLE
[Distributeur] Name component shouldn't be an h2

### ISSUE THREAD

**Issue by MartinWeb** [AxaFrance/design-system#1938]
The Name Header component display an h2 on the DOM and is placed at the very top of the page. 
But after this Name, we always use an HeaderTitle which displays an h1.

The problem is that in accessibility there should be an order for headings, meaning the h1 should always come first, then possibles h2s, then h3s, etc.

We should remove the h2 on the Name component and use a normal text instead, and keep its design.

**This would be a breaking change with the test in the consumer, so it should be inside a 2.0 version.**
